recipes/halotools/meta.yaml

```diff
@@ -0,0 +1,106 @@
+{% set name = "halotools" %}
+{% set version = "0.5" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "7f5e8c3afcee35baa053a2c084d1b1301de70a285fce2391b7e46fedb53e041d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy 1.9.*  # [not (win and py36)]
```

Comment:
I'm not sure if having the `build` versions of `numpy` constrained like this without similar constraints on the `run` section is going to cause issues. You could end up in a situation where on non-windows and py36 systems, you build with 1.9 but then run with 1.12. 

Typically we'd use `numpy x.x` here, but I'm not familiar with the specific constraints of this package.